saleor/graphql/translations/descriptions.py

```diff
@@ -0,0 +1,3 @@
+class TranslationFieldDecriptions:
```

Comment:
We can make it slightly shorter:

```suggestion
class TranslationDescriptions:
```